<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">body {
  background: linear-gradient(135deg, #5D6D83 0%, #9EB3C0 100%);
  font-family: 'Roboto', sans-serif;
  color: #ffffff;
}

.container {
  background: rgba(36, 52, 70, 0.9); /* Adjusted for better text contrast */
  backdrop-filter: blur(5px);
  border-radius: 10px;
  padding: 20px;
  box-shadow: 0 4px 6px rgba(36, 52, 70, 0.3); /* Color picked from the palette */
}

.button {
  background-color: #28344C;
  color: white;
  border: none;
  padding: 10px 20px;
  border-radius: 5px;
  cursor: pointer;
  transition: background-color 0.3s ease;
}

.button:hover {
  background-color: #5D6D83; /* Lighter color from the palette for hover effect */
}

.input-field {
  background: rgba(46, 56, 77, 0.2); /* Adjusted for input field visibility */
  border: 1px solid #28344C; /* Using color from palette for border */
  padding: 10px;
  border-radius: 5px;
  color: white;
}

/* Additional styles for other UI elements can be added here */
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser { background-color: rgba(255, 255, 255, 0); color: black; }
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;Username:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>-30</y>
      <width>951</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0%); /* Fully transparent background */
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;title&gt;Scrummy 1.0&lt;/title&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:21pt; font-weight:700; font-style:italic;&quot; bgcolor=&quot;transparent&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Segoe UI','sans-serif'; font-size:14px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:72pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI','sans-serif'; font-size:72pt; color:#ffffff;&quot;&gt;Scrummy 1.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>600</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2D2D2D; /* Dark grey background */
    color: white; /* White text */
    border-radius: 5px; /* Rounded corners */
    padding: 10px 20px; /* Padding around text */
    font-size: 18px; /* Font size */
    border: none; /* No border */
    text-align: center; /* Center text */
    text-decoration: none; /* No text decoration */
    display: inline-block; /* Display as inline-block */
    transition: all 0.3s; /* Smooth transition for hover effects */
    cursor: pointer; /* Show hand cursor on hover */
}

QPushButton:hover {
    background-color: #3A3A3A; /* Slightly lighter grey on hover */
    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2); /* Subtle shadow effect on hover */
}

QPushButton:pressed {
    background-color: #1A1A1A; /* Even lighter grey for pressed state */
}
</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerbutton_main">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>510</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2D2D2D; /* Dark grey background */
    color: white; /* White text */
    border-radius: 5px; /* Rounded corners */
    padding: 10px 20px; /* Padding around text */
    font-size: 18px; /* Font size */
    border: none; /* No border */
    text-align: center; /* Center text */
    text-decoration: none; /* No text decoration */
    display: inline-block; /* Display as inline-block */
    transition: all 0.3s; /* Smooth transition for hover effects */
    cursor: pointer; /* Show hand cursor on hover */
}

QPushButton:hover {
    background-color: #3A3A3A; /* Slightly lighter grey on hover */
    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2); /* Subtle shadow effect on hover */
}

QPushButton:pressed {
    background-color: #1A1A1A; /* Even lighter grey for pressed state */
}
</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signIn_Button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>510</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2D2D2D; /* Dark grey background */
    color: white; /* White text */
    border-radius: 5px; /* Rounded corners */
    padding: 10px 20px; /* Padding around text */
    font-size: 18px; /* Font size */
    border: none; /* No border */
    text-align: center; /* Center text */
    text-decoration: none; /* No text decoration */
    display: inline-block; /* Display as inline-block */
    transition: all 0.3s; /* Smooth transition for hover effects */
    cursor: pointer; /* Show hand cursor on hover */
}

QPushButton:hover {
    background-color: #3A3A3A; /* Slightly lighter grey on hover */
    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2); /* Subtle shadow effect on hover */
}

QPushButton:pressed {
    background-color: #1A1A1A; /* Even lighter grey for pressed state */
}
</string>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsbutton_main">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>480</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>440</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser { background-color: rgba(255, 255, 255, 0); color: black; }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:12pt; font-weight:700; font-style:normal;&quot; bgcolor=&quot;transparent&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI','sans-serif'; font-size:16pt; font-weight:400; color:#ffffff;&quot;&gt;Password:&lt;/span&gt;&lt;span style=&quot; font-size:16pt; font-weight:400; color:#ffffff;&quot;&gt; &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_username">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>390</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #CCCCCC; /* Light grey border */
    border-radius: 4px; /* Slightly rounded corners for a modern look */
    padding: 5px; /* Padding inside the text field */
    background-color: #FFFFFF; /* White background color */
    color: #000000; /* Black text color */
    selection-background-color: #4A90E2; /* Blue selection color */
    selection-color: #FFFFFF; /* White selection text color */
}

QLineEdit:focus {
    border-color: #4A90E2; /* Blue border color when the QLineEdit has focus */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_password">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #CCCCCC; /* Light grey border */
    border-radius: 4px; /* Slightly rounded corners for a modern look */
    padding: 5px; /* Padding inside the text field */
    background-color: #FFFFFF; /* White background color */
    color: #000000; /* Black text color */
    selection-background-color: #4A90E2; /* Blue selection color */
    selection-color: #FFFFFF; /* White selection text color */
}

QLineEdit:focus {
    border-color: #4A90E2; /* Blue border color when the QLineEdit has focus */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="parentboardButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>90</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Parentboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backlog_test">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>150</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Backlog Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adminButton">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2D2D2D; /* Dark grey background */
    color: white; /* White text */
    border-radius: 5px; /* Rounded corners */
    padding: 10px 20px; /* Padding around text */
    font-size: 18px; /* Font size */
    border: none; /* No border */
    text-align: center; /* Center text */
    text-decoration: none; /* No text decoration */
    display: inline-block; /* Display as inline-block */
    transition: all 0.3s; /* Smooth transition for hover effects */
    cursor: pointer; /* Show hand cursor on hover */
}

QPushButton:hover {
    background-color: #3A3A3A; /* Slightly lighter grey on hover */
    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2); /* Subtle shadow effect on hover */
}

QPushButton:pressed {
    background-color: #1A1A1A; /* Even lighter grey for pressed state */
}
</string>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
   <widget class="QLabel" name="bg_main">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>711</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets/resources.qrc">:/MainWindowbg.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="boder_login">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>541</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser {
    border: 2px solid #C0C0C0; /* Light grey border */
    border-radius: 8px; /* Rounded corners */
    background-color: rgba(255, 255, 255, 10%); /* Semi-transparent white background */
    color: #333; /* Dark grey text */
    padding: 10px; /* Space inside the borders */
    selection-background-color: #3874f2; /* Blue selection background */
    selection-color: #ffffff; /* White text for selection */
    font-family: 'Segoe UI', sans-serif; /* Modern font family */
    font-size: 14px; /* Adequate font size */
}

QTextBrowser::hover {
    border: 2px solid #808080; /* Darker border on hover */
}

QTextBrowser:focus {
    border: 2px solid #275E92; /* Blue border for focus */
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;title&gt;Scrummy 1.0&lt;/title&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI','sans-serif'; font-size:14px; font-weight:700; font-style:italic;&quot; bgcolor=&quot;transparent&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ForgetPass_Button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>570</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2D2D2D; /* Dark grey background */
    color: white; /* White text */
    border-radius: 5px; /* Rounded corners */
    padding: 10px 20px; /* Padding around text */
    font-size: 18px; /* Font size */
    border: none; /* No border */
    text-align: center; /* Center text */
    text-decoration: none; /* No text decoration */
    display: inline-block; /* Display as inline-block */
    transition: all 0.3s; /* Smooth transition for hover effects */
    cursor: pointer; /* Show hand cursor on hover */
}

QPushButton:hover {
    background-color: #3A3A3A; /* Slightly lighter grey on hover */
    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2); /* Subtle shadow effect on hover */
}

QPushButton:pressed {
    background-color: #1A1A1A; /* Even lighter grey for pressed state */
}
</string>
    </property>
    <property name="text">
     <string>Forget Password</string>
    </property>
   </widget>
   <zorder>bg_main</zorder>
   <zorder>boder_login</zorder>
   <zorder>textBrowser_2</zorder>
   <zorder>textBrowser</zorder>
   <zorder>quitButton</zorder>
   <zorder>registerbutton_main</zorder>
   <zorder>signIn_Button</zorder>
   <zorder>settingsbutton_main</zorder>
   <zorder>textBrowser_3</zorder>
   <zorder>input_username</zorder>
   <zorder>input_password</zorder>
   <zorder>parentboardButton</zorder>
   <zorder>backlog_test</zorder>
   <zorder>adminButton</zorder>
   <zorder>logo</zorder>
   <zorder>ForgetPass_Button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
